src/main/webapp/i18n/de/global.json

```diff
@@ -276,7 +276,8 @@
             "maxbytes": "Dieses Feld sollte nicht mehr als {{max}} bytes haben.",
             "pattern": "Dieses Feld muss das Muster {{pattern}} erfüllen.",
             "number": "Dieses Feld muss eine Zahl sein.",
-            "datetimelocal": "Dieses Feld muss eine Datums- und Zeitangabe enthalten."
+            "datetimelocal": "Dieses Feld muss eine Datums- und Zeitangabe enthalten.",
+            "validPattern": "Dieses Feld beinhaltet einen ungültigen Muster"
```

Comment:
see below (invalidPattern?)
It should be: "Dieses Feld beinhaltet ein ungültiges Muster."
Or what I would find better "Dieses Feld entspricht nicht dem vorgegebenen Muster." As you either define valid patterns that must be included instead of invalid patterns that are checked against.